messenger/custom-transport.rst

```diff
@@ -108,20 +108,19 @@ Here is a simplified example of a database transport::
         public function send(Envelope $envelope): Envelope
         {
             $encodedMessage = $this->serializer->encode($envelope);
-            $uuid = Uuid::uuid4()->toString();
```

Comment:
You should keep $uuid, otherwise it will generate a new one for `TransportMessageIdStamp` no ?